ReadMe.md

```diff
@@ -129,6 +130,41 @@ That's exactly how we generate our `full` builds.
 
 Finally, you will have **`firmware/.obj/f7/full.dfu`** file that can be distributed and flashed.
 
+## MacOS
+### Prerequisites
+1. Install `gcc-arm-embedded`
+```
+brew install --cask gcc-arm-embedded
+```
+
+2. Install `open-ocd` if you want use debugger
+```
+brew install open-ocd
+```
+
+3. Build hex2dfu from source. Clone repo, build executable file and copy it to `PATH`
+```
+git clone git@github.com:rusdacent/hex2dfu.git
+cd hex2dfu
+gcc hex2dfu.c ED25519/*.c -o hex2dfu
+sudo cp hex2dfu /usr/local/bin
+```
+
+### Compile firmware
+```
+make -j$(nproc)
```

Comment:
`nproc` is not available on mac os

Reply:
O_o
![image](https://user-images.githubusercontent.com/5871715/140730759-adfc3b6c-d9e3-4ab1-9990-5428c4e5d3f2.png)